sw/supervision/pc_aircraft.ml

```diff
@@ -104,6 +105,25 @@ let gcs_or_edit = fun file ->
   | 2 -> ignore (Sys.command (sprintf "%s -edit '%s'&" gcs file))
   | _ -> failwith "Internal error: gcs_or_edit"
 
+let checkout_git_version = fun sha ->
+  (Sys.command (sprintf "git checkout '%s'" sha));
```

Comment:
Yes indeed. Moreover you will then be in detached head mode, which might freak out some people. And a make will be needed so you need to go to the prompt anyway. A copy SHA could be better, but that didn't work out in OCAML. Any tips?